### ISSUE TITLE
Adding lamin C antibody

### ISSUE THREAD

**Issue by Fabien-Maury** [obophenotype/human-phenotype-ontology#10398]
**Preferred term label:**
lamin C antibody positivity

**Synonyms**
prelamin C antibody positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against human lamin C.
(PMID:1959878|PMID:1290474)

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:536|ORPHA:2137|ORPHA:2582

**Your nano-attribution (ORCID)**
0000-0003-2854-0167

**Comment by pnrobinson** [comment]
@Fabien-Maury 
Adding new term: 
- ID: HP:6000644
- label: Anti-lamin C antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:1290474|PMID:1959878
- def: The presence of autoantibodies in the blood circulation that react against human lamin C.
- comment: 
- orcid: orcid:0000-0003-2854-0167
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10398
- synonyms: Prelamin C antibody positivity(NONE)
